controller/src/main/java/cern/jarrace/controller/domain/Entrypoint.java

```diff
@@ -0,0 +1,54 @@
+package cern.jarrace.controller.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author timartin
+ * Domain class that represents a registered {@link Entrypoint}
+ */
+
+@Entity
+@Table(name = "ENTRYPOINT_TABLE")
+public class Entrypoint {
+
+    /**
+     * Unique identifier for the {@link Entrypoint}
+     */
+    @Id
+    @Column(name = "AGENT_NAME", unique = true)
```

Comment:
for now I removed all the JPA annotations since we are not using them anymore, we are keeping track of things in conventional lists